spec/cancan/ability_spec.rb

```diff
@@ -361,6 +361,20 @@ class A; include B; end
     expect(@ability.can?(:read, A)).to be(true)
   end
 
+  it "allows to check ability in a decorated class" do
+    class ADecorator < SimpleDelegator
+      def kind_of?(klass)
```

Comment:
Yeah, I agree with your view, but that's how actually draper works regarding this case, isn't it? I just replicated was Draper was doing ... since SimpleDelegator and Draper work in different ways, what should we do here? Check for `defined?(:__get_object__)` or `.object`? Not cool either ... kind of lost here.